.gitignore

```diff
@@ -1,5 +1,6 @@
 database.py
 config.txt
+settings.py
```

Comment:
what is this file for? :black_cat:

Reply:
It was for decouple stuff, but I since moved it 😅  it's deleted now!